### PR TITLE
added addHelp() method and docs

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by OskarStark** [comment]
ping @core23 @sagikazarmark

File: Form/FormMapper.php [hunk 1]
```diff
@@ -200,9 +200,22 @@ public function create($name, $type = null, array $options = array())
     public function setHelps(array $helps = array())
     {
         foreach ($helps as $name => $help) {
-            if ($this->admin->hasFormFieldDescription($name)) {
-                $this->admin->getFormFieldDescription($name)->setHelp($help);
-            }
+            $this->addHelp($name, $help);
```
Comment (OskarStark) [inline Form/FormMapper.php]: this is not necessary, we could keep the old logic here, but IMO its better to rely on one logic

**Comment by sagikazarmark** [comment]
Not sure I see the added value. What's the rationale?

**Comment by core23** [comment]
Good idea :+1:

**Comment by OskarStark** [comment]
@sagikazarmark see the docs in this PR, i added my use case there

**Comment by sagikazarmark** [comment]
Ah, missed it, cool. :+1: 

Maybe help could be added directly using field options in the `add` method?

**Comment by OskarStark** [comment]
you can added it directly, but then you can't use it in the Extension :smile:

**Comment by OskarStark** [comment]
ping @rande

**Pull request merged**